devsetup/Makefile

```diff
@@ -174,6 +174,34 @@ edpm_deploy_instance: ## Spin a instance on edpm node
 edpm_play_cleanup: ## Cleanup EDPM openstackansibleee resource
 	-oc delete openstackansibleee deploy-external-dataplane-compute
 
+.PHONY: standalone
+standalone: export EDPM_COMPUTE_NAME=standalone
+standalone: export EDPM_COMPUTE_VCPUS=8
+standalone: export EDPM_COMPUTE_RAM=20
+standalone: export EDPM_COMPUTE_DISK_SIZE=70
+standalone: edpm_compute ## Create standalone VM
+	$(eval $(call vars))
+	scripts/standalone.sh ${EDPM_COMPUTE_SUFFIX}
+
+.PHONY: standalone_cleanup
+standalone_cleanup: export EDPM_COMPUTE_NAME=standalone
+standalone_cleanup: edpm_compute ## Delete standalone VM
+	$(eval $(call vars))
+	scripts/edpm-compute-cleanup.sh ${EDPM_COMPUTE_SUFFIX}
+
+.PHONY: standalone_snapshot
+standalone_snapshot: export EDPM_COMPUTE_NAME=standalone
+standalone_snapshot: ## Create standalone snapshot
+	$(eval $(call vars))
+	virsh --connect=qemu:///system detach-device-alias ${EDPM_COMPUTE_NAME} fs0 --live
+	virsh --connect=qemu:///system snapshot-create-as --atomic --domain ${EDPM_COMPUTE_NAME} --name clean
+
+.PHONY: standalone_revert
+standalone_revert: export EDPM_COMPUTE_NAME=standalone
+standalone_revert: ## Revert standalone snapshot
+	$(eval $(call vars))
+	virsh --connect=qemu:///system snapshot-revert --domain ${EDPM_COMPUTE_NAME} --name clean
```

Comment:
I've included snapshot/revert

Reply:
The revert command should have `--domainname` instead of `--name` like in https://github.com/openstack-k8s-operators/data-plane-adoption/pull/57